### ISSUE TITLE
feature: add optional attribute to only start process, when required processes reached RUNNING state.

### ISSUE THREAD

**Issue by michaelhammann** [Supervisor/supervisor#1450]
For my use case it would be useful to only start a process after dependent process reaches RUNNING state if the optional process config parameter depends_on is used. Starting processes consecutively using a priority is not enough in my case. 
This feature requires the dependent process to be set to RUNNING state correctly and therefore depends on #1446.

I am using supervisor 4.2.2. and I also created a suggestion for the implementation here: #1449

Appreciate your help!

**Comment by mnaberez** [comment]
Duplicate of #122
